src/main/java/io/sealights/plugins/sealightsjenkins/buildsteps/cli/SealightsCLIBuildStep.java

```diff
@@ -33,6 +42,66 @@ public SealightsCLIBuildStep(boolean enabled, boolean failBuildIfStepFail,
         this.cliRunner = cliRunner;
     }
 
+    /* * The goal of this method is to support migration of data between versions
+    * of this plugin.
+    */
+    private Object readResolve() {
+        SealightsCLIBuildStep _this = this;
+        if (cliRunner != null) {
+            _this = resolveFromCLiRunner();
+        }
+        if(commandMode instanceof CommandMode.ConfigView){
+            _this =resolveTechOption();
+        }
+        return _this;
+    }
+
+    private SealightsCLIBuildStep resolveTechOption(){
+        String packagesIncluded = ((CommandMode.ConfigView) commandMode).getPackagesIncluded();
```

Comment:
Cast the CommanMode only once - no need to repeat '((CommandMode.ConfigView) commandMode)' over and over.